app/src/main/java/com/example/poste/activities/EditPostActivity.java

```diff
@@ -71,6 +72,16 @@ protected void onCreate(Bundle savedInstanceState) {
             postTitle.setText(currentPost.getTitle());
             postDescription.setText(currentPost.getDescription());
             postLink.setText(currentPost.getUrl());
+            if (!currentPost.getTags().isEmpty()) {
+                String tagsNewText = "";
+                for (String s : currentPost.getTags()) {
+                    if (tagsNewText.length() > 0) {
+                        tagsNewText = tagsNewText + ",";
```

Comment:
From a UI POV, have a space after a comma is much preferred

Reply:
Change to use ", "